kinbot/kb.py

```diff
@@ -126,15 +130,13 @@ def main():
         logging.error('Error with initial structure optimization.')
         return
     if any(well0.freq[i] <= 0 for i in range(len(well0.freq))):
-        logging.error('Found imaginary frequency for initial structure.')
-        return
+        logging.error('Found imaginary frequency (freq # {}, value = {}) for initial structure.'.format(i, well0.freq[i]))
 
```

Comment:
Occassionaly a structure has a single small imaginary freq. related to a Me rotation or some other small movement that the optimizer cannot eliminate. So instead of completely killing the entire calculation it continues running but the error is noted in the log file. If the frequency is detrimental to the calculation then the user can check, but  the only time I have seen this is when the error comes from a small imaginary freq that is related to a Me group or something similar.